### DISCUSSION TITLE
Greyhawk

### DISCUSSION THREAD

**Discussion by larrykid** [triplea-game/triplea#8402] [Feature Requests & Ideas]
Can someone make a new Greyhawk map and more mobs? Middle Earth mods are not  bad but the maps are ugly as hell,

**Comment by DanVanAtta** [comment]
@larrykid you may want to post this in: https://forums.triplea-game.org/category/28/map-making

The map maker communities does not tend to watch the github issue & discussion list.

**Comment by larrykid** [comment]
I like the games ,these guys that make them either dont play it themselves to see the mistakes in the AI or are just bad players themselves. I'm playing Iron War as the axis because most of all of these games if you play the allies you will trounce them. Anyway to make it more playable I added 15 bucks to each major ally US, British,  Russia and 10 to the other 9 allies...round 8 it's over. The problems are easy to see. The US cant make up its mind and doesn't build boats at full tilt to get Germany's fleet out of the water. The US does what every axis has allies AI does buys planes . China in iron war protects the first base close to the coast and leaves the capital and everything behind it wide open, they wont fall back. Same with Russia they will not leave the base ose to Finland and you can just drive around them ,once japan kills those arty and trucks Russia starts with it's over. So people that makes these play them?
Sent from Yahoo Mail on Android 
 
  On Sun, Dec 20, 2020 at 12:50 PM, Dan Van Atta<notifications@github.com> wrote:   


@larrykid you may want to post this in: https://forums.triplea-game.org/category/28/map-making

The map maker communities does not tend to watch the github issue & discussion list.

**Comment by DanVanAtta** [comment]
Valuable feedback @larrykid . I don't think the right people will see this here though. Map specific chatter is best on the forum.
